.github/workflows/ci-test.yml

```diff
@@ -383,6 +416,7 @@ jobs:
       - lint
       - unit-test
       - e2e-test
+      - docker-compose-test
```

Comment:
not good enough, look at the job itself.   you should echo out the status, and add it to the big boolean && expression below.

Reply:
oi completely missed it, sorry about that